src/golang/cmd/server/handler/list_workflow_objects.go

```diff
@@ -23,7 +23,7 @@ import (
 //		`api-key`: user's API Key
 // Response:
 //	Body:
```

Comment:
What's the difference between Response: [something] vs Response: Body: [something]?

Reply:
They are the same :) Updated some comments so that all are consistent (have `Body`)

Reply:
Encountered some linting issues on spacing, I'm merging with the old version without add `Body`s. Could come back to this after we got linting issues resolved